### PR TITLE
Various packaging fixes

### PR DESCRIPTION
**Pull request by ericstj**
Fix the annoying \r\n from our package descriptions
Update the baseline list.
Don't warn when folks have opted to skip missing files.
Set targetpath so that binclashlogger can catch package bin-clashes.

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets [hunk 1]
```diff
@@ -890,7 +891,7 @@
       <_SkipCreatePackage Condition="'$(SkipCreatePackageOnMissingFiles)' == 'true' AND '@(_missingFiles)' != ''">true</_SkipCreatePackage>
     </PropertyGroup>
 
-    <Warning Condition="'$(_SkipCreatePackage)' == 'true'" Text="Skipping package creation for $(NuSpecPath) because the following files do not exist: @(_missingFiles)" />
+    <Message Condition="'$(_SkipCreatePackage)' == 'true'" Text="Skipping package creation for $(NuSpecPath) because the following files do not exist: @(_missingFiles)" />
```
Comment (weshaggard) [inline src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets]: :)

File: src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets [hunk 2]
```diff
@@ -746,8 +747,8 @@
 
     <PropertyGroup>
       <Description Condition="'$(UseRuntimePackageDescription)' == 'true' AND '$(RuntimeDisclaimer)' != ''">$(RuntimeDisclaimer)</Description>
-      <Description Condition="'$(UseRuntimePackageDescription)' != 'true' AND '$(RuntimeDisclaimer)' != ''">$(RuntimeDisclaimer) \r\n $(Description)</Description>
-      <Description Condition="'@(SyncInfoLines)' != ''">$(Description) \r\n %(SyncInfoLines.Identity)</Description>
+      <Description Condition="'$(UseRuntimePackageDescription)' != 'true' AND '$(RuntimeDisclaimer)' != ''">$(RuntimeDisclaimer) %0A$(Description)</Description>
+      <Description Condition="'@(SyncInfoLines)' != ''">$(Description) %0A%(SyncInfoLines.Identity)</Description>
```
Comment (weshaggard) [inline src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets]: Not exactly sure how this works but my hope is that it puts a correct newline in the package description.

**Comment by weshaggard** [comment]
LGTM

File: src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets (hunk 2 above)
Comment (chcosta) [inline src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets]: Cool, I was going to look at this issue today (time permitting).

**Comment by chcosta** [comment]
LGTM

**Pull request merged**